chain/index/chain-indexing-overview-for-rpc-providers.MD

```diff
@@ -60,34 +71,137 @@ The ChainIndexer [periodically runs](https://github.com/filecoin-project/lotus/b
 [Fevm]
 EthTxHashMappingLifetimeDays = 0
 
+TODO: https://github.com/filecoin-project/lotus/pull/12450#discussion_r1792743250
+> see note in #12421, this one is duplicated with a moved redirect
+
 [Events]
 DisableHistoricFilterAPI = false
 DatabasePath = ""
 ```
 
-## Upgrade Steps
+## Upgrade
+
+### Preparation
+> **Note:** One can upgrade/downgrade between [pre-ChainIndexer](#previous-indexing-system) and [with-ChainIndexer](#chainindexer-indexing-system) Lotus versions without conflict because they persist state to different directories and don't rely on each other. No backup is necessary (but extra backups don't hurt). There is still a [backfilling step though when downgrading](#downgrade-steps).
+
+These upgrade steps assume one has multiple nodes in their fleet and can afford to have a node not handling traffic potentially for days potentially per [backfill timing below](#backfill-timing).
+
+One should also check to ensure they have [sufficient disk space](#backfill-disk-space-requirements).
+
+### Part 1: Create a Backfilled ChainIndexer `chainindex.db`
+1. **Route traffic away from an initial node**
+   - Example: prevent a load balancer from routing traffic to a designated node.
+2. **Stop the designated Lotus Node**
+   - Stop the designated Lotus node before starting the upgrade and backfill process.
+3. **Update Configuration**
+   - Modify the Lotus configuration to enable the `ChainIndexer` as described in the [`ChainIndexer Config` section above](#chainindexer-config). 
+4. **Restart Lotus Node**
+   - Restart the Lotus node with the new configuration.
+   - The `ChainIndexer` will begin indexing **real-time chain state changes** immediately in the `${LOTUS_HOME}/chainindex` directory.
+   - *However, it will not automatically index any historical chain state (i.e., any previously existing chain state prior to the upgrade).*
+5. **Backfill**
+   - See the ["Backfill" section below](#backfill).
+   - This could potentially take days per [Backfill Timing](#backfill-timing).
+6. **Ensure node health**
+   - Perform whatever steps are usually done to validate a node's health before handling traffic (e.g., log scans, smoke tests)
+7. **Route traffic to the backfilled node that is now using ChainIndexer**
+8. **Ensure equal or better correctness and performance**
+   - ChainIndexer-using nodes should have full correctness and better performance when compared to [pre-ChainIndexer](#previous-indexing-system) nodes.
+
+### Part 2: Update Other Nodes
+Now that one has a `${LOTUS_HOME}/chainindex/chaindinex.db` from a trusted node, it can be copied to additional nodes to expedite bootstrapping.
```

Comment:
s/chaindinex/chainindex - there are a few occurrences of this misspelling in here

Reply:
oh, also, s/LOTUS_HOME/LOTUS_PATH, there's a lot of those in this doc, LOTUS_HOME isn't a thing in the Lotus codebase but it'll respect LOTUS_PATH if you set it

Reply:
Doh - my bad.  Both have been fixed.  Resolving.